index.html

```diff
@@ -16,7 +16,7 @@
    <header>
     <nav class="navbar">
         <div class="hamburger">
-            <img src="./assets/ic_menu.svg" alt="hamburger">
+            <img class="hamb" src="./assets/ic_menu.svg" alt="hamburger">
         </div>
```

Comment:
- [ ] Good job @adeola003, Please here notice that the icon is almost hidden on a `375px` display and I have to scroll to see the whole icon. Please make sure to add some padding and margin as visible on the figma design so the user has a better experience using your application. 

### Your design 
![image](https://user-images.githubusercontent.com/69077061/223527077-3d2fb69a-4b5c-47bd-ad63-b47e36e1a7a4.png)

### Figma design 
![image](https://user-images.githubusercontent.com/69077061/223527487-dfc9cd17-7f12-4811-b42f-4c6ada1d0619.png)